### ISSUE TITLE
Record as Samples

### ISSUE THREAD

**Issue by abinrosemathew** [naudio/NAudio#410]
I need to record input of my Mic and record as 1 minute wave files.The 1 minute file saving should be done while recording.

Any helps?

Please

**Comment by markheath** [comment]
Yes, quite easy, just use WaveIn and in the DataAvailable event handler, write the recorded data into a WaveFileWriter. if the size of the current WAV file is 1 minute, then close it, and write create a new WaveFileWriter.

**Comment by abinrosemathew** [comment]
Thanks a lot Mark.

:)
